cannot write data from mysql to hive

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.


### What happened

This parameter [SQL] in the hive sink parameter is required. If this parameter is not added, it will appear, but I don't see the description in the official document. After adding this parameter, the table that needs to insert data in the SQL statement will pop up an error and can't find the corresponding table if it is not created in hive. Logically, it should be created without this table? Or do you need to build a table in advance before you can use it?

### SeaTunnel Version

2.0.5 dev

### SeaTunnel Config

```conf
This is a hive sink without SQL parameters configured:
sink{
    Hive{
        source_table_name = "mytable2"
        result_table_name = "test.seatunnel_test_tbl_v1"
        partition_by = ["dt"]
        save_mode = "append"
    }
}
This is the hive sink for which the SQL parameter must be configured:
sink{
    Hive{
        source_table_name = "mytable2"
        result_table_name = "test.seatunnel_test_tbl_v1"
        partition_by = ["dt"]
        save_mode = "append"
        sql = "insert into test.seatunnel_test_tbl_v1 partition(dt) select * from mytable2"
    }
}
```


### Running Command

```shell
1
```


### Error Exception

```log
ERROR seatunnel.Seatunnel: Plugin[org.apache.seatunnel.spark.sink.Hive] contains invalid config, error: please specify [sql] as non-empty 


Caused by: org.apache.spark.sql.catalyst.analysis.NoSuchTableException: Table or view 'seatunnel_test_tbl_v1' not found in database 'test';
```


### Flink or Spark Version

spark-2.4.8

### Java or Scala Version

jdk1.8

### Screenshots

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/seatunnel-connectors/seatunnel-connector-spark-hive/src/main/scala/org/apache/seatunnel/spark/sink/Hive.scala
+++ b/seatunnel-connectors/seatunnel-connector-spark-hive/src/main/scala/org/apache/seatunnel/spark/sink/Hive.scala
@@ -29,7 +29,11 @@ import org.apache.spark.sql.{DataFrameWriter, Dataset, Row}
 class Hive extends SparkBatchSink with Logging {
 
   override def checkConfig(): CheckResult = {
-    CheckConfigUtil.checkAllExists(config, "sql", "source_table_name", "result_table_name")
+    if (config.hasPath("sql")) {
+      CheckResult.success()
+    } else {
+      CheckConfigUtil.checkAllExists(config, "source_table_name", "result_table_name")
+    }
   }
 
   override def prepare(env: SparkEnvironment): Unit = {}

